test.sh

```diff
@@ -8,6 +8,12 @@
 
 set -e
 
+# This adds backwards compatibility if only single version needs to be testing
+# In CI we would like to test single version but VERSIONS= means, that nothing is tested
+# make test TARGET=<OS> VERSIONS=<something> ... checks single version for CLI
+# make test TARGET=<OS> SINGLE_VERSION=<something> ... checks single version from Testing Farm
+VERSIONS=${SINGLE_VERSION:-$VERSIONS}
```

Comment:
Is this necessary?
Wouldn't it suffice to set just `VERSIONS=$VERSION` in the TMT plan? Then no changes would be needed in container-common-scripts.

Reply:
Good point. But if the `VERSION` is not specified then `VERSIONS` will be empty and nothing is tested.
The VERSIONS are specified in Makefile and the TMT plan overrides it. And therefore we have a problem.

Reply:
Thanks. I did not catch it from the comment, but now I understand.